index.html

```diff
@@ -73,7 +68,7 @@ <h1 class="sr-only">
               <a class="nav-link" href="#">Contact Us</a>
             </li>
           </ul>
-          <button>Donate Now</button>
+          <button class="donate_now">Donate Now</button>
```

Comment:
Hi Abed, you need to go through all of the HTML and where neccesary replace the `<div>` tags with semantic tags such as `<header>`, `<footer>`, `<section>`, `<article>` etc